src/duckling/entitysystem/entity-system.service.ts

```diff
@@ -72,6 +72,16 @@ export class EntitySystemService {
     }
 
     /**
+     * Renames an entity
+     * @param  oldEntityKey The old key of the entity to be renamed
+     * @param  newEntityKey The new key of the entity to be renamed
+     * @param  mergeKey     Used to merge updates.
+     */
+    renameEntity(oldEntityKey : EntityKey, newEntityKey : EntityKey, mergeKey? : any) {
```

Comment:
Should we unit test the entirety of the entity-system.service? We don't have any yet